src/infrastructure/http/deps.py

```diff
@@ -0,0 +1,17 @@
+from fastapi import Request
+
+from apps.shared.errors import ForbiddenError
+from infrastructure.http.domain import MindLoggerHeaders
+
+
+async def mindlogger_headers(request: Request) -> MindLoggerHeaders:
+    try:
+        mindlogger_content_source = MindLoggerHeaders(
+            mindlogger_content_source=request.headers.get(
+                "mindlogger-content-source"
+            )
+        )
+    except Exception:
+        raise ForbiddenError
```

Comment:
Moreover, it should not raise en error if there is no such field

Reply:
We will change it after the alignment with the frontend team when the whole implementation will be ready. Currently the regular approach is used